cli/src/commands/mod.rs

```diff
@@ -128,6 +129,8 @@ enum Command {
     Squash(squash::SquashArgs),
     Status(status::StatusArgs),
     #[command(subcommand)]
+    Tag(tag::TagSubcommand),
```

Comment:
```rust
#[command(subcommand)]
Branch(branch::BranchSubcommand),

#[command(subcommand)]
Config(config::ConfigSubcommand),
```

I use `cli/src/commands/branch.rs` as a reference. Do you think small refactoring for constant naming??

Reply:
Sure, if you don't mind. I don't think it's required for this PR; I mainly care that it's consistent within this file. If we do make the naming consistent across files, we could go with either "Command" or "Subcommand" as suffix. Browsing a few pages of https://github.com/search?q=clap%3A%3ASubcommand&type=code and grepping for "command {" suggests it's pretty even between them. I would prefer changing `branch.rs` to use "Command" in that case because it's a smaller change. But I don't feel strongly either way.